lib/include/kframework/avm/teal/teal-driver.md

```diff
@@ -377,9 +380,9 @@ If X is a byte-array, it is interpreted as a big-endian unsigned integer. bitlen
     requires lengthBytes(B) <=Int MAX_BYTEARRAY_LEN
      andBool Bytes2Int(B, BE, Unsigned) >Int 0
 
-  rule <k> bitlen => panic(INVALID_ARGUMENT) ... </k>
-       <stack> (B:Bytes) : _ </stack>
-    requires lengthBytes(B) >Int MAX_BYTEARRAY_LEN
+//  rule <k> bitlen => panic(INVALID_ARGUMENT) ... </k>
+//       <stack> (B:Bytes) : _ </stack>
+//    requires lengthBytes(B) >Int MAX_BYTEARRAY_LEN
```

Comment:
```suggestion
```
I am thinking this is not needed, since we should check that integer values are valid before pushing them onto the stack.